### PR TITLE
Updated database migration

### LINKED ISSUE #255: DB migration handling
Cloudlogs database has changed a bit over the years, but the https://github.com/magicbug/Cloudlog/tree/master/application/migrations folder wasn't kept up-to-date. 

The assets/install.sql file contains an (older version) database dump which is executed. It's not up-to-date and doesn't contain the migrations table, so that will directly lead to failures on new installations.

Apart from use as a reference - is the sql/tables folder used anywhere? 

My suggestion/plan for work would be: 
Install a current cloudlog build with the assets/install.sql file, create the migrations table there. 
Then I'd go ahead, build migrations up to the current database version.

Maybe the sql/tables-Folder shouldn't be used at all? 

What do you think about the idea to clean this up? Would such a PR be accepted?

### PR DESCRIPTION
**Pull request by Manawyrm**
(This commit needs to be merged after the JSON stuff)

This does a lot of stuff again, fixes #255. 
tl;dr: It fixes the DB issues for existing users and new installations.

I have updated the install.sql to a recent state (with migration-table present). 
New installations start of with a proper database layout and work fine. (and will handle future migrations gracefully).
The sql/tables folder was removed. 
A new (big) migration will update any old database to the current state. I have tested this with 2 old installations of mine, which (after a git pull) were properly upgraded to the current database layout. This also creates the new tables for station_profile and such. 

The 014_migration.php isn't an example for picture perfect code, but it can be used as a reference for future migrations :)

### CODE REVIEW

**Comment by Manawyrm** [comment]
This then fixes #255.

**Comment by magicbug** [comment]
Merged the json stuff, I'm seeing a conflict for "sql/tables/cat.sql" so can't merge the lazy way, but will look at it after dinner.

**Comment by Manawyrm** [comment]
That's an interesting merge conflict, seeing as this commit should delete the file :D 
If it makes you too much trouble, send a message, then I'll quickly fix this on my side.

**Comment by Manawyrm** [comment]
Hm, why was this closed by github?

**Comment by Manawyrm** [comment]
Anyway: This can now be merged online :)

**Comment by magicbug** [comment]
Interestingly that time i commented on a merge it closed an issue! bonkers
